commons/env/schlapp-mac.env

```diff
@@ -24,3 +24,6 @@ export SOLR_PORT=9546
 export SOLR_MEMORY=8g
 export PGBINDIR="/Applications/Postgres.app/Contents/Versions/10/bin/"
 export SSH_USER="cmpich"
+export NODE_ENV="development"
+export ZFIN_ADMIN="cmpich@zfin.org"
+export WIKI_HOST="devwiki.zfin.org"
```

Comment:
What is devwiki.zfin.org?

Reply:
That is an old configuration parameter before we moved the wiki to the cloud. It is not used any longer and needs to be globally removed. Until then, it needs to be set.